qipr_approver/approver/forms/about_you_form.py

```diff
@@ -43,12 +43,11 @@ def __init__(self, user=User()):
                               'div_classes': 'about__field--width100'}
 
         self.self_classification = {'name': 'self_classification',
-                           'label': 'Self Classification',
-                           'placeholder': 'If you selected yes, enter your training program here.',
```

Comment:
Initially no, but it just crossed my mind why I want it back. Changes coming soon